components/script/dom/node.rs

```diff
@@ -619,6 +620,10 @@ impl Node {
         window_from_node(self).client_rect_query(self.to_trusted_node_address())
     }
 
+    pub fn style(&self) -> Option<Arc<ComputedValues>> {
```

Comment:
This should probably move to `Element`, since this getting called on, e.g., a text-node, will panic the layout thread.

Maybe worth doing before it lands, maybe fine as a followup? A doc comment here would be nice too :)